src/CONST.ts

```diff
@@ -86,6 +86,9 @@ const CONST = {
     DEFAULT_TABLE_NAME: 'keyvaluepairs',
     DEFAULT_ONYX_DUMP_FILE_NAME: 'onyx-state.txt',
     DEFAULT_POLICY_ROOM_CHAT_TYPES: [chatTypes.POLICY_ADMINS, chatTypes.POLICY_ANNOUNCE, chatTypes.DOMAIN_ALL],
+    DEFAULT_MAX_EXPENSE_AMOUNT_NO_RECEIPT: 10000000000,
+    DEFAULT_MAX_EXPENSE_AMOUNT: 10000000000,
+    DEFAULT_MAX_EXPENSE_AGE: 10000000000,
```

Comment:
isn't it the same as the ones from policy section of constants

```
        DEFAULT_MAX_EXPENSE_AGE: 90,
        DEFAULT_MAX_EXPENSE_AMOUNT: 200000,
        DEFAULT_MAX_AMOUNT_NO_RECEIPT: 2500,
```

Reply:
I renamed the prefix of the added consts from `DEFAULT_` to `EMPTY_`